features/networking/sriov.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Sriov related scenarios
+
+  # @author zzhao@redhat.com
+  # @case_id OCP-29944
+  @admin
+  Scenario: sriov operator can be setup and running well
+    Given I switch to cluster admin pseudo user
+    And I use the "openshift-sriov-network-operator" project
+    And I wait up to 120 seconds for the steps to pass:
+    """
+    And all existing pods are ready with labels:
```

Comment:
as @pruan-rht  mentioned in https://github.com/openshift/verification-tests/pull/1027#discussion_r417732484

these steps have an existing `15 * 60` timeout, do we need the `wait up to 120 seconds` loop?

Reply:
thanks @rbbratta  updated.